### PR TITLE
Change OWNERS of amp-consent to wg-analytics & wg-ads

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

**Comment by lannka** [comment]
@dvoytenko to approve as @zhouyx is OOO

**Comment by lannka** [comment]
@jridgewell as @dvoytenko is also out

File: extensions/amp-consent/OWNERS [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
   rules: [
     {
-      owners: [{name: 'zhouyx'}, {name: 'torch2424'}],
+      owners: [{name: 'ampproject/wg-ads'}, {name: 'ampproject/wg-analytics'}],
```
Comment (rcebulko) [inline extensions/amp-consent/OWNERS]: Is removing `torch2424` intentional?

Comment (lannka) [reply extensions/amp-consent/OWNERS]: yes, he is not working on this anymore.

**Pull request merged**